### PR TITLE
chore: BUN-2587 improve setup docs

### PR DESCRIPTION
**Pull request by bc-marco**
## What?

Improve setup docs for local development

## Why?

There are unclear steps and requirements, also some values are not aligned with the current app

## Testing / Proof

N/A

## How can this change be undone in case of failure?

Undo this PR

@bigcommerce/b2b-buyer-portal

### CODE REVIEW

**Review by jp06 — COMMENTED** [review]
This should still be in progress, but I hope you don't mind some comments/suggestions. Feel free to dismiss.

File: README.md [hunk 1]
```diff
@@ -151,20 +156,24 @@ Environment variables have been updated so you can run your UI directly into pro
   window.B3 = {
     setting: {
       store_hash: '{{settings.store_hash}}',
-      channel_id: 1, // this will vary depending on your store channel id
+      channel_id: <your-channel-id>>, // override this depending on your store channel id, you can check this on Channel Manager -> select the desired channel, and the url will show: https://store-<storehash>.mybigcommerce.com/manage/channel/<your-channel-id>/overview
```
[the remark this answers is not in the document]
Comment (bc-marco) [reply README.md:163]: Interesting, I tried this before and it didn't work, but I tried again and seems that is working now, I'll update the PR

File: README.md [hunk 2]
```diff
@@ -107,18 +107,23 @@ For assistance with activating the remote buyer portal or to inquire about multi
 4. Copy environment variables: `cp apps/storefront/.env-example apps/storefront/.env`.
 5. Update the following values in `.env`:
 
-- `VITE_ASSETS_ABSOLUTE_PATH`: For deployment, set this to the absolute path of the hosted compiled assets.
+- `VITE_ASSETS_ABSOLUTE_PATH`(Optional): For custom host, set this if you want to host by yourself the compiled assets after your run `yarn build`command. ie. https://my.custom.cdn/generated/b2b/assets
+
+  It will point to the absolute path where will be hosted
```
[the remark this answers is not in the document]
Comment (bc-marco) [reply README.md]: Thanks for your suggestion!

File: README.md [hunk 3]
```diff
@@ -239,6 +250,7 @@ Also, you'll have to input the following header script:
 
 ### Common issues:
 
+- **Stencil CLI** Currently we're working to bring full support to integrate buyer portal into that tool, there are issues on loging out and cart management, if you find other problems feel free to open an issue report.
```
Comment (icatalina) [inline README.md:253]:
```suggestion
- **Stencil CLI** Currently we're working to bring full support to integrate buyer portal into [stencil-cli](https://developer.bigcommerce.com/docs/storefront/stencil), there are issues on logging out and cart management, if you find other problems feel free to open an issue report.
```

File: apps/storefront/.env-example [hunk 4]
```diff
@@ -11,13 +11,13 @@ VITE_TRANSLATION_SERVICE_URL=https://api-b2b.bigcommerce.com
...
 
 # Set this to TRUE to debug in your default storefront
 VITE_LOCAL_DEBUG=FALSE
 
 # URL where the GraphQL is hosted, usually the same one as B2B_URL_API. If the GraphQL API is hosted locally, set this to the local URL
 VITE_LOCAL_GRAPHQL_ORIGIN=https://api-b2b.bigcommerce.com
 
-# For building the app, set this to the absolute path where the compiled assets will be hosted
-VITE_ASSETS_ABSOLUTE_PATH="<YOUR_ASSETS_ABSOLUTE_PATH>"
+# For custom host, set this to the absolute path where will be hosted the generated assets after your run yarn build
```
Comment (deov31) [inline apps/storefront/.env-example:22]:
```suggestion
# For custom host, set this to the absolute path where will be hosted the generated assets after your run `yarn build`
```
🍹

**Review by deov31 — APPROVED** [review]

**Comment by bc-marco** [comment]
@icatalina ♻️

**Review by icatalina — APPROVED** [review]

**Pull request merged**